features/autorisatie/personen/gezag-persoon-beperkt/adressering/autorisatie.feature

```diff
@@ -66,3 +66,64 @@ Regel: Vragen met fields om een adres in adressering wanneer de gebruiker niet g
     | adressering                       | 81110 81120 81130 81140 81150 81160 81170 81210 81330 81340 81350 PANM03 PANM04 PANM05 PANM06 PAVP01 PAVP02 PAVP03 PAVP04 | land (81310)         |
     | adresseringBinnenland             | 81110 81120 81130 81140 81150 81160 81170 81210 PANM03 PANM04 PANM05 PANM06 PAVP01 PAVP02 PAVP04                          | adresregel1 (PAVP03) |
     | adresseringBinnenland             | 81110 81120 81130 81140 81150 81160 81170 81210 PANM03 PANM04 PANM05 PANM06 PAVP01 PAVP02 PAVP03                          | adresregel2 (PAVP04) |
+
+Regel: Een afnemer geautoriseerd voor de virtuele rubriek 'adressering' (AD.01) mag alle adres velden van adressering opvragen
+
+  Abstract Scenario: Afnemer vraagt om <fields> en is geautoriseerd voor de virtuele rubriek 'adressering'
+    Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
+    | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+    | 81180 AD01                      | N                        | 20201128                |
+    En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+    | naam      | waarde |
+    | afnemerID | 000008 |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                             | waarde                                  |
+    | type                             | ZoekMetAdresseerbaarObjectIdentificatie |
+    | adresseerbaarObjectIdentificatie | 0599010051001502                        |
+    | fields                           | <fields>                                |
+    Dan heeft de response 0 personen
+
+    Voorbeelden:
+    | fields                            |
+    | adressering                       |
+    | adressering.adresregel1           |
+    | adressering.adresregel2           |
+    | adressering.adresregel3           |
+    | adressering.land                  |
+    | adresseringBinnenland             |
+    | adresseringBinnenland.adresregel1 |
+    | adresseringBinnenland.adresregel2 |
+
+Regel: Een afnemer geautoriseerd voor de virtuele rubriek 'elektronische adressering' (AD.02) mag geen adres velden van adressering opvragen
```

Comment:
dit suggereert dat dit echt gebruik verbiedt. Wat als een afnemers zowel AD.02 heeft als PANM03? Mag die dan geen aanhef vragen?

Reply:
Moet er nog op de specifieke adressering rubrieken zoals PANM03 worden geautoriseerd?

Reply:
Ik vrees van wel. Tot nu toe is er geen enkele afnemer die voor adressering of elektronische adressering geautoriseerd is. Maar wel voor aantal gegevens die daaronder vallen..

Reply:
> Je mag bijvoorbeeld adressering.aanhef vragen als je PANM03 hebt of als je PAAD01 of PAAD02 hebt. En je mag adressering.adresregel1 vragen als je PAVP03 hebt of als je PAAD01 hebt.
> 
> Als je PAAD02 en PAVP03 hebt, mag je aanhef, aanschrijfwijze, gebruikInLopendeTekst en adresregel1 vragen, niet adresregel2, adresregel3 of land.

Hiermee zegt je dus dat je het oude autorisatie rubrieknummers kan combineren met de nieuwe autorisatie rubrieknummers. Hiermee wordt dus nieuwe situaties geïntroduceerd die niet mogelijk zijn als de oude autorisatie rubrieknummers zijn uitgefaseerd. Dat gaat neem ik aan toch gebeuren?